bench/parser.go

```diff
@@ -98,7 +98,7 @@ func (p *Parser) readBenchmark(line string) (*Benchmark, error) {
 
 	// runs - doesn't include units
 	tmp, split = internal.Popleft(split)
-	if bench.Runs, err = strconv.Atoi(tmp); err != nil {
+	if bench.Runs, err = strconv.Atoi(tmp); err != nil && !strings.Contains(line, "Intel(R)") {
```

Comment:
```suggestion
    // Ignore CPU information for now, until we support parsing it: https://github.com/bobheadxi/gobenchdata/issues/47
	if bench.Runs, err = strconv.Atoi(tmp); err != nil && !strings.Contains(line, "cpu: ") {
```

Reply:
We can ignore all cpu information, not just Intel CPUs

Reply:
you suggest checking for "cpu" in line?